Source/Android/app/build.gradle

```diff
@@ -89,3 +92,7 @@ dependencies {
     // Allows FRP-style asynchronous operations in Android.
     compile 'io.reactivex:rxandroid:1.2.1'
 }
+
+def getVersion() {
+    return 'git describe --abbrev=0'.execute([], project.rootDir).text.trim()
```

Comment:
version number is the user displayed string, whereas version code is what the os uses to determine whether theres been an upgrade. Version code is current hard-coded to 13, no idea why